tests/unit/cocoon/test_utilities.py

```diff
@@ -1949,6 +1949,65 @@ def test_default_fx_forward_model(
         )
         self.assertEqual(mapping_gt, mapping_test, msg="mapping not correctly remapped")
 
+    @parameterized.expand(
+        [
+            (
+                    "No fx forward transactions present.",
+                    pd.DataFrame(
+                        data=[
+                            [1000, 1, 1, -500, "GBP", "NonFxType1", "b1", "2020/01/01"],
+                            [1001, 1, 1, 500, "GBP", "NonFxType2", "sl", "2020/01/01"],
+                            [1002, 1, 1, -500, "USD", "NonFxType3", "~", "2020/01/01"],
+                        ],
+                        columns=[
+                            "TX_ID",
+                            "Price",
+                            "price (local)",
+                            "quantity",
+                            "currency",
+                            "type",
+                            "leg",
+                            "date",
+                        ],
+                    ),
+                    {
+                        "transactions": {
+                            "required": {
+                                "code": "$fund_id",
+                                "settlement_date": "date",
+                                "total_consideration.amount": "quantity",
+                                "total_consideration.currency": "currency",
+                                "transaction_currency": "currency",
+                                "transaction_date": "date",
+                                "transaction_id": "TX_ID",
+                                "transaction_price.price": "Price",
+                                "transaction_price.type": "$Price",
+                                "type": "type",
+                                "units": "quantity",
+                            }
+                        }
+                    },
+                    ValueError
+            )
+        ]
+    )
+    def test_default_fx_forward_model_on_missing_fx_transaction_type(
```

Comment:
This test looks like its generalised for any expected exception, so maybe leave the test name as :
`test_default_fx_forward_model_failed()`

we have a (rough) convention of naming test cases wehere we have expected success and failed tests:
`test_format_holdings_response_success`
`test_format_instruments_response_fail`

when the test runs, unittests will append the current testcase name to the output, eg:

Test passed: `test_default_fx_forward_model_failed_No_fx_forward_transactions_present` 
Test passed: `test_default_fx_forward_model_failed_some_testcase_1`
Test passed: `test_default_fx_forward_model_failed_some_testcase_2` 
etc etc ...

Reply:
Updated the test names to match the pattern used in rest of the tests within the class.  The success and failure cases are now `test_default_fx_forward_model_success` and `test_default_fx_forward_model_failure`